mod_reforged/hooks/entity/tactical/humans/bounty_hunter.nut

```diff
@@ -39,5 +39,46 @@
 	{
 		__original();
 		::Reforged.Skills.addPerkGroupOfEquippedWeapon(this, 4);
+
+		if (this.getBodyItem() != null)
+		{
+			local armor = this.getBodyItem();
+			local armorAttachment = null;
+
+			if (armor.getConditionMax() < 115)
+				{
+					if (::Math.rand(1, 100) <= ::Reforged.Config.ArmorAttachmentChance.Tier3)
+					{
+						armorAttachment = ::Reforged.ItemTable.ArmorAttachmentNorthern.roll({
+							Apply = function ( _script, _weight )
+							{
+								local conditionModifier = ::ItemTables.ItemInfoByScript[_script].ConditionModifier;
+								if (conditionModifier > 20) return 0.0;
+								return _weight;
+							}
+						})
+					}
+				}
+
+			else if (armor.getConditionMax() >= 115)
```

Comment:
The `else if` condition here is redundant as there are only two possible situations: either it is `< 115` or it is not. So this line can simply be `else`